tutorials/ui/gui_containers.rst

```diff
@@ -179,3 +179,30 @@ to its rect size:
     func set_some_setting():
         # Some setting changed, ask for children re-sort
         queue_sort()
+
+ .. code-tab:: csharp
+
+    using Godot;
+    using System;
+
+    public class Container : Godot.Container
+    {
+        public override void _Notification(int what)
+        {
+            if (what == NotificationSortChildren)
+            {
+                // Must re-sort the children
+                foreach (Control c in GetChildren())
```

Comment:
I know this variable is called `c` in GDScript too, but maybe we could change it and not endorse one letter variable naming? Doesn't seem like a good practice in general.